docs/pages/install/windows.mdx

```diff
@@ -0,0 +1,82 @@
+# Install Pelican on Windows
+
+This document explains how to install Pelican on Windows.
+
+## Install Pelican as a standalone binary
+
+Pelican provides a binary file instead of a `MSI` installer for Windows. You need to download and extract the binary and add the binary to your `PATH` environment variable.
+
+1. Navigate to [Pelican download page](../install.mdx#download-pelican-binary) and select the Pelican Windows binary.
+
+2. In the **Operating System** section, select **Windows**. Copy the link to `pelican_Windows_x86_64.zip`.
+
+3. Download the file to your computer and extract the binary from the `.zip` file.
+
+4. You may run the binary in the extracted folder, but it is recommended that you add Pelican binary to your `PATH` environment variable to allow `pelican` to be called directly from your PowerShell. To run Pelican binary from the extracted folder:
+
+    ```bat
+    rem Go to the downloaded Pelican folder
+    $ cd D:\Downloads\pelican_Windows_x86_64
+
+    rem Run Pelican binary
+    $ pelican.exe --version
+
+    Version: 7.5.8
+    Build Date: 2024-03-01T18:13:00Z
+    Build Commit: d260a07d3b057d19b7fdd36125f91a8768531258
+    Built By: goreleaser
+    ```
+
+## Add Pelican binary to your `PATH`
+
+* Add Pelican binary to your `PATH` for the current PowerShell
+
+  ```bat
+  rem Go to the binary folder
+  $ cd pelican_Linux_arm64
+
+  rem Add current folder to the PATH
+  $ set PATH=%PATH%;%CD%
```

Comment:
This did not work for me. I do not use PowerShell, so I don't know what the correct command is. Could also be a version issue. I'm using

```
Edition	Windows 11 Pro
Version	23H2
Installed on	‎2/‎14/‎2023
OS build	22631.3593
Experience	Windows Feature Experience Pack 1000.22700.1003.0
```

and

```
PS C:\Users\a_owe\Downloads> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.22621.2506
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22621.2506
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```